spec/models/amusement_park_spec.rb

```diff
@@ -3,5 +3,6 @@
 RSpec.describe AmusementPark, type: :model do
   describe "relationships" do
     it { should have_many(:rides) }
+    it { should have_many(:mechanics).through(:rides) }
   end
```

Comment:
We are missing a model test for `AmusementPark#mechanic_names`

Reply:
Always run your model tests separately from your feature tests - `bundle exec rspec spec/models` - that way SimpleCov will tell you if it's < 100%, and you can see what you're missing.